package.json

```diff
@@ -15,7 +15,7 @@
     "jstreegrid": "3.9.4",
     "magicsuggest-alpine": "1.0.0",
     "ngeo": "2.4.0-daily.20180716T092301Z.76f80c4.HEAD",
-    "openlayers": "https://api.github.com/repos/openlayers/openlayers/tarball/f284b95cb897b3ce1b49b985b58ebe1ca76be1fe",
+    "openlayers": "4.6.5",
```

Comment:
use the latest 5.x release instead ? `5.1.3`

Reply:
No, this is for the admin interface...